src/plugins/irc-events/link.js

```diff
@@ -49,7 +49,10 @@ function parse(msg, url, res, client) {
 	case "text/html":
 		var $ = cheerio.load(res.text);
 		toggle.type = "link";
-		toggle.head = $("title").text();
+		toggle.head =
+			$("meta[property=\"og:title\"]").attr("content")
+			|| $("title").text()
+			|| "No title found.";
```

Comment:
I would actually reword this in `Untitled page` rather than `No title found.` The latter displays like an error message while the former presents a fact about the page. Nitpick, I know.

And because I know this is meant to be consistent with `No description found.`, I would simply remove it. It looks sloppy. If there is no description, let's not show any, it's leaner and doesn't make things obfuscated either.